src/gg.regression.unity.experimental.runespawn/Editor/Scripts/EditorCommandServer.cs

```diff
@@ -163,7 +163,18 @@ private static void HandleClient(object obj)
 
             if (Enum.TryParse(commandStr, true, out CommandType commandType))
             {
-                EditorCommand editorCommand = new EditorCommand(commandType, client);
+                EditorCommand editorCommand;
+                if (commandType == CommandType.Start)
+                {
+                    // Read the optional bot sequence path
+                    string botSequencePath = Utilities.ReadLine(stream);
```

Comment:
Does this throw an error if there is no line? Or does it return null?

Reply:
Good catch, my bad for missing this. Added the check here.